news/_posts/public-oss-demo/2021-06-30-public-oss-demo-6-2021.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "PrestaShop Public OSS Demo - June 2021 session"
+subtitle: "We held the sixth session of the open source project's public demo"
+date:   2021-06-31
+authors: [ PrestaShop ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's sixth monthly public demo session was held and [streamed live](https://www.youtube.com/watch?v=Flyqo21eICg) on June 30, 2021.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=Flyqo21eICg) Beginning of the session, introduction
+- [00:34](https://www.youtube.com/watch?v=Flyqo21eICg&t=34) Presentation of the agenda
+- [01:14](https://www.youtube.com/watch?v=Flyqo21eICg&t=74) Event rules
+- [02:28](https://www.youtube.com/watch?v=Flyqo21eICg&t=148) Project announcements
+- [04:53](https://www.youtube.com/watch?v=Flyqo21eICg&t=293) Demonstration #1 - How to develop a module 100% multistore compatible by [@marionf](https://github.com/marionf) and [@matthieu-rolland](https://github.com/matthieu-rolland)
+- [25:25](https://www.youtube.com/watch?v=Flyqo21eICg&t=1525) Demonstration #2 - Introduce DKIM signing for emails by [@Hlavtox](https://github.com/Hlavtox)
+- [47:10](https://www.youtube.com/watch?v=Flyqo21eICg&t=2830) Demonstration #3 - Upgrade to Symfony 4 by [@atomiix](https://github.com/atomiix)
+- [58:38](https://www.youtube.com/watch?v=Flyqo21eICg&t=3518) Closure
+
+
+You can use the timestamps to jump directly to the parts you want to watch.
+
+[Watch the replay on YouTube](https://www.youtube.com/watch?v=Flyqo21eICg){: .btn .btn-lg .btn-info}
+{: .text-center}
+
+Special thanks to [@Hlavtox](https://github.com/Hlavtox) who is a regular contributor of the PrestaShop project and joined us today to present his work!
+
+## Next session in July
+
+The next session will be streamed live on [**Wednesday, July 28, 2021 at 4pm CEST**](https://www.youtube.com/watch?v=pE6lVtdsIhE).
```

Comment:
```suggestion
We noticed that 2pm CEST was a little early in the morning for people watching us live from the Americas, so we decided that future sessions will start two hours later — at 4pm. We hope this will encourage our American community to watch the demo live and discuss with us :)

The next session will be streamed live on [**Wednesday, July 28, 2021 at 4pm CEST**](https://www.youtube.com/watch?v=pE6lVtdsIhE). See you then!
```